### DISCUSSION TITLE
part-ii/2020-5/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#122] [Question comments]
Part II 2020 Algebraic Topology: Paper 2, Section II, F 

https://questions.tripos.org/part-ii/2020-5/

**Comment by artie2000** [comment]
Let $F_2=<a,b|>\equiv\pi_1(S_1\vee S_1)$.

<details>
<summary>(b): Map </summary>

$\alpha: S^1\to S^1\vee S^1$ by $\overline{a^3b^{-7}}$.

</details>

(c): One possibility is $\langle a,b^2,b^{-1}ab\rangle$.
